diagnostic_feedback/public/js/src/student/student_quiz.js

```diff
@@ -0,0 +1,239 @@
+function StudentQuiz(runtime, element) {
+
+    var studentQuiz = this;
+    studentQuiz.startOver = false;
+    studentQuiz.movingToStep = false;
+
+    /* Javascript for Student view in LMS.*/
+
+    var common = new Common(),
+        form = $(".diagnostic-feedback #student_view_form"),
+        studentViewFormSelector = ".diagnostic-feedback #student_view_form",
+
+        //selectors
+        currentAnswerContainer = ".diagnostic-feedback .current",
```

Comment:
With all these selectors, if you always wrap them in `$(...)` it would be better just to declare them as `var $currentAnswerContainer = $(".diagnostic-feedback .current");` in the first place.